### PR TITLE
Master v2

### PR DESCRIPTION
**Pull request by KobeDelobelle**
merge from masterv2 into master

### CODE REVIEW

File: PetShopV2/PetShopV2/Views/LoginPage.xaml [hunk 1]
```diff
@@ -11,4 +11,21 @@
             <Button VerticalOptions="Center" Text="Login" Command="{Binding LoginCommand}" />
         </StackLayout>
     </ContentPage.Content>
+
+
+    <!--<StackLayout VerticalOptions="Center" Margin="20">
```
Comment (MichWouters) [inline PetShopV2/PetShopV2/Views/LoginPage.xaml:16]: Please remove commented code from code.

File: PetShopV2/PetShopV2/Views/AboutPage.xaml.cs [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 namespace PetShopV2.Views
 {
     public partial class AboutPage : ContentPage
     {
+
         public AboutPage()
         {
             InitializeComponent();
         }
+
```
Comment (MichWouters) [inline PetShopV2/PetShopV2/Views/AboutPage.xaml.cs:13]: Please remove unnecessary whitespace

**Review by MichWouters — CHANGES_REQUESTED** [review]
Made some small corrections

File: PetShopV2/PetShopV2/Views/LoginPage.xaml (hunk 1 above)
Comment (KobeDelobelle) [reply PetShopV2/PetShopV2/Views/LoginPage.xaml:16]: i fixed it

File: PetShopV2/PetShopV2/Views/AboutPage.xaml.cs (hunk 2 above)
Comment (KobeDelobelle) [reply PetShopV2/PetShopV2/Views/AboutPage.xaml.cs:13]: codemaid will do it for me
